tokio-trace/src/lib.rs

```diff
@@ -548,19 +548,19 @@ macro_rules! event {
             }
         }
     });
-    (target: $target:expr, $lvl:expr, { $( $k:ident = $val:expr ),* }, $($arg:tt)+ ) => ({
+    (target: $target:expr, $lvl:expr, { $( $k:ident = $val:expr $(,)* ),* }, $($arg:tt)+ ) => ({
```

Comment:
It's worth noting that this will allow _any number_ of trailing commas, so something like 
```rust
event!(
   tokio_trace::Level::DEBUG, 
   foo = 1, 
   baz = 2,
   ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
);
```
is also valid.

I'm not sure if we actually _care_ about that, though...

Reply:
Good point. I can change the repetition token to `?` to allow 0 or 1.

Reply:
Addressed [here](https://github.com/tokio-rs/tokio/pull/891#issuecomment-462527704)